manifests/pam.pp

```diff
@@ -131,6 +133,17 @@
         }
       }
 
+      #only allow root and members of the group wheel to su
+      if $only_root_may_su {
```

Comment:
This `if` clause should be left out here - if you switch back `only_root_may_su` to `false`, the file won't get an update.